### ISSUE TITLE
Rest API HEAD error

### ISSUE THREAD

**Issue by MarcHagen** [shlinkio/shlink#711]
#### How Shlink is set-up

* Shlink Version: 2.1.2
* PHP Version: ?
* How do you serve Shlink: Docker image (stable)
* Database engine used: Docker image

#### Summary
When doing a HEAD request to the `/rest/health` endpoint we get a "500 error" on traefik.
On further investigation it has to do with the returning headers. Because there are **2** Content-Length headers.

I also saw that the Rest API is not returning data on HEAD requests, i think i will make a PR to allow that.

#### Current behavior

```
> HEAD /rest/health HTTP/1.1
> Host: 10.10.0.31:6080
> User-Agent: curl/7.69.1
> Accept: */*
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 404 Not Found
< Content-Type: application/problem+json
< X-Request-Id: e1c5f309-833a-417b-a9fc-b633e93b5aa7
< Content-Length: 107
< Server: swoole-http-server
< Connection: keep-alive
< Date: Sun, 05 Apr 2020 19:32:30 GMT
< Content-Length: 107
```

#### Expected behavior

```
> HEAD /rest/health HTTP/1.1
> Host: 10.10.0.31:6080
> User-Agent: curl/7.69.1
> Accept: */*
>
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< Content-Type: application/health+json
< X-Request-Id: e1c5f309-833a-417b-a9fc-b633e93b5aa7
< Content-Length: 107
< Server: swoole-http-server
< Connection: keep-alive
< Date: Sun, 05 Apr 2020 19:32:30 GMT
```

#### How to reproduce

1. `docker run --name shlink -p 8080:8080 -e SHORT_DOMAIN_HOST=doma.in -e SHORT_DOMAIN_SCHEMA=https shlinkio/shlink:stable`
2. `curl -I -L -v http://shlink:8080/rest/health -o /dev/null`

**Comment by acelaya** [comment]
Thanks for letting me know. I will investigate where is that coming from.

**Comment by acelaya** [comment]
@MarcHagen There was an issue when handling `HEAD`requests. It should be fixed once I release v2.1.3

**Comment by acelaya** [comment]
@MarcHagen I have just released v2.1.3 which fixes this issue.

**Comment by MarcHagen** [comment]
Perfect! Thank you, works like a charm
